### DISCUSSION TITLE
华硕RT-AC68U的梅林固件386.3_2无法安装aliyundrivewebdav-merlin-arm384-armv7.tar.gz

### DISCUSSION THREAD

**Discussion by pennywu2016** [messense/aliyundrive-webdav#255] [Q&A]
麻烦帮忙看看，谢谢。提示如下：
【2022年01月15日 17:37:42】: ========================== step 1 ===============================
【2022年01月15日 17:37:42】: 开启插件离线安装！
【2022年01月15日 17:37:43】: 检测到你上传的离线安装包：aliyundrivewebdav-merlin-arm384-armv7.tar.gz，安装包大小: 2.1M
【2022年01月15日 17:37:43】: 安装包md5sum校验值：78ebe8fa38633d672fc366906e918e30
【2022年01月15日 17:37:43】: 尝试解压离线安装包离线安装包...
【2022年01月15日 17:37:43】: 解压错误，错误代码：1
【2022年01月15日 17:37:43】: 估计是错误或者不完整的的离线安装包！
【2022年01月15日 17:37:43】: 删除相关文件并退出！
【2022年01月15日 17:37:43】: 删除文件：/tmp/aliyundrivewebdav-merlin-arm384-armv7.tar.gz
【2022年01月15日 17:37:44】: ============================ end ================================

**Comment by messense** [comment]
帮不了，没有梅林固件的路由器。试了 aliyundrivewebdav-merlin-arm384-armv7.tar.gz 这个文件是正常的。

**Comment by pennywu2016** [comment]
好的，谢谢回复。




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "messense/aliyundrive-webdav"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年1月15日(星期六) 下午5:49
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [messense/aliyundrive-webdav] 华硕RT-AC68U的梅林固件386.3_2无法安装aliyundrivewebdav-merlin-arm384-armv7.tar.gz (Discussion #255)





 
帮不了，没有梅林固件的路由器。试了 aliyundrivewebdav-merlin-arm384-armv7.tar.gz 这个文件是正常的。
 
—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Reply by icfnight** [reply]
68U要装380版本

**Reply by 1044615793** [reply]
有没有最新版68u可用的，老版本已经不支持了
